pkg/upgrade/upgrade.go

```diff
@@ -603,3 +607,58 @@ func GetDeployedRelease(ctx context.Context, cli client.Client) (cluster.Release
 	// could be a clean installation or both CRs are deleted already
 	return cluster.Release{}, nil
 }
+
+func cleanupNimIntegrationTechPreview(ctx context.Context, cli client.Client, oldRelease cluster.Release, applicationNS string) error {
+	var errs *multierror.Error
+
+	if oldRelease.Version.Minor >= 14 && oldRelease.Version.Minor <= 15 {
+		logger := logf.FromContext(ctx)
```

Comment:
nit:
can we name it as log , just for consistency across the code

Reply:
Resolved in https://github.com/opendatahub-io/opendatahub-operator/pull/1369/commits/610ea2e16fe15e42a163648bec5a7529d06438bb.